.eslintrc

```diff
@@ -6,7 +6,8 @@
   "rules": {
     "curly": ["error", "multi-line"],
     "prettier/prettier": ["error", { "trailingComma": "es5" }],
-    "no-case-declarations": "error"
+    "no-case-declarations": "error",
+    "max-len": ["error", { "ignoreStrings": true }]
```

Comment:
I'd just wrap the string with concatenation.

Reply:
Okay :( just felt gross